rest/resources/rest.json

```diff
@@ -81,5 +81,11 @@
     }
   },
 
-  "numBlocksTransactionFeeStats": 300
+  "numBlocksTransactionFeeStats": 300,
+
+  "deployment": {
+    "deploymentTool": "",
```

Comment:
The reference rest.json should include the "N/A" or "Manual" recommended values. Bootstrap will put their own